recipes/protobufc/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "protobufc" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/protobuf-c/protobuf-c/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d4cb022d55f49796959b07a9d83040822e39129bc0eb28f4e8301da17d758f62
+  patches:
+    # Fix up CMake config https://github.com/protobuf-c/protobuf-c/pull/719
+    - patches/0001-squash-clean-cmake.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - ninja
+  host:
+    - libprotobuf
+    - libabseil
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libprotobuf-c${SHLIB_EXT}
```

If we're building for windows, you'll need to split this. Also, there's a binary `protoc-gen-c` being packaged that we should test.
```suggestion
    - test -f ${PREFIX}/lib/libprotobuf-c${SHLIB_EXT}   # [unix]
    - if not exist %LIBRARY_BIN%/protobuf-c.dll         # [win]
    - protoc-gen-c --version
```

---

added as suggested.